jenkins/e2e-image/e2e-runner.sh

```diff
@@ -432,6 +432,12 @@ elif [[ ! "${JOB_NAME}" =~ -pull- ]]; then
   fi
 fi
 
+if [[ -n "${PRIORITY_PATH:-}" ]]; then
+  export PATH="${PRIORITY_PATH}:${PATH}"
+fi
+echo "kubectl on path is:"
+kubectl version --client
```

Comment:
It's something we print if the cluster comes up (and we collect the server version then), but this is recording it prior to that. I can omit it if you'd like, or promise to come back around and clean it up in e2e.go.